Misc/NEWS.d/next/Library/2019-08-29-20-26-08.bpo-30988.b-_h5O.rst

```diff
@@ -0,0 +1 @@
+Fix parsing of emails with invalid address headers having a leading or trailing dot. Patch by tsufeki.
```

Comment:
For what it's worth, the real name of the patch author is usually used in the news entry (though it's not mandatory).